sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -15,13 +15,7 @@
 
 // Map of UR error codes to PI error codes
 static pi_result ur2piResult(zer_result_t urResult) {
-
-  // TODO: replace "global lifetime" objects with a non-trivial d'tor with
-  // either pointers to such objects (which would be allocated and dealocated
-  // during init and teardown) or objects with trivial d'tor.
-  // E.g. for this case we could have an std::array with sorted values.
-  //
-  static std::unordered_map<zer_result_t, pi_result> ErrorMapping = {
+  std::unordered_map<zer_result_t, pi_result> ErrorMapping = {

```

Comment:
Is this change going to add performance overhead?

Reply:
It certainly will, but 1) it should be negligible and 2) it is temporary until we switch to UR.
There is already a map lookup overhead before this change, not this map is re-constructed too.
I'd let the testing see if it causes any real perf issues

Reply:
I see, thanks.